ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -93,6 +93,27 @@ keys:
           description: 'Action as string
 
             Example: "permit GSHUT 65123:123"'
+  daemons:
+    type: list
+    primary_key: name
+    convert_types:
+    - dict
+    display_name: Custom Daemons
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          required: true
+          display_name: Daemon Name
+        exec:
+          type: str
+          required: true
+          description: 'command to run as a daemon
+
```

Comment:
can we make the description value consistent, I see here we did
``` 
description: 'string

`
```

and in the next file it's

```
description: |
     string
```

I think the second one looks better no?

Reply:
This line is autogenerated and only read by other scripts.

Reply:
right. This is autogenerated file